build-output-api/isr-revalidation/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "isr-revalidation",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {
+    "node-fetch": "2.6.0"
```

Comment:
To populate the node modules for the `revalidate` function, I installed `node-fetch` here and moved the resulting `node_modules` directory into the `revalidate` function. This works well enough for the example, but if anyone has a better suggestion for constructing this, let me know.

Reply:
Could just use vanilla `http.request()` if you think importing `fetch` is overkill.

Reply:
Updated!